app/md-data-table/scripts/mdSelect.js

```diff
@@ -83,8 +83,23 @@ function mdSelect($compile) {
       checkbox.attr('ng-click', '$mdSelect.toggle($event)');
       checkbox.attr('ng-checked', '$mdSelect.isSelected()');
       checkbox.attr('ng-disabled', '$mdSelect.disabled');
+
+      var $elem = angular.element('<td class="md-cell md-checkbox-cell">');
```

Comment:
I would name this temporary variable `cell` instead of `$elem`. I try to reserve the `$` prefix for services and private variables.